### ISSUE TITLE
Kernel Panic on Boot After 23 January 2018 Security Update for Sierra

### ISSUE THREAD

**Issue by mickelsn** [theracermaster/Gigabyte-GA-Z77X-DSDT-Patch#87]
## System Information

 - OS: macOS 10.12.6
 - Script: GA-Z77X macOS Install v2.1.0
 - MB: GA-Z77X-UP5 TH
 - CPU: Intel Core i7-3770K
 - IGPU: Intel HD Graphics 4000
 - dGPU: GeForce GTX 650 Ti

## Issue Type

 - [ ] I'm trying to boot the macOS installer
 - [X] I'm trying to boot my macOS install

 - [ ] I'm encountering errors during/after the post-install script

 - [ ] Can you add support for my board?

## Issue Description

Well, after much success in just doing an upgrade for macOS Security Updates whenever needed, I finally got bitten today. I installed Security Update 2018-001 Sierra on my Hackintosh, foolishly without a backup, and I now get consistent kernel panics on boot. Image of the panic being written to screen over the boot logo is below. Anyone have any ideas?

![img_2465](https://user-images.githubusercontent.com/15273114/35316267-b1750d34-009e-11e8-91dd-9c86493c8b84.png)

**Comment by mickelsn** [comment]
I did just look at this a bit more closely, and it might be panicking after loading up the Lilu kernel extension. Appears that it's running Lilu 1.1.6, and I note here that the extension is now up to 1.2.1...but that 1.2.0 broke compatibility with some Lilu add-ons and I'm not sure which ones are used in this package...

Again, ideas always welcomed.

**Comment by yanneg** [comment]
I have the exact same issue. Short of restoring to an old backup, I'm not sure how to recover from this.

**Comment by al3xtjames** [comment]
It's the old version of Lilu that's causing this. To boot, you can add `-liluoff`, which you prevent Lilu from loading.

**Comment by al3xtjames** [comment]
It's the old version of Lilu that's causing this. To boot, you can add `-liluoff`, which you prevent Lilu from loading.

**Comment by mickelsn** [comment]
Thanks, Alex, I'll try that later tonight once I'm back at my machine. Any ideas on a permanent solution to this besides disabling Lilu? I'll edit my clover config file to lock this in for now, but if I recall Lilu is being used for a couple of things in your package like enabling purchased iTunes video playback...

**Comment by al3xtjames** [comment]
Updating Lilu (and its plugins) is all that you need to do.

**Comment by mickelsn** [comment]
OK, thanks for the help -- this worked! Here's what I ended up doing, in case it helps others:

1. Booted with `-liluoff` flag in Clover.
2. Mounted EFI partition and navigated to `EFI/CLOVER/kexts/Other`. Noted all kexts that were there and their versions.
3. Checked out the [Lilu GutHub](https://github.com/vit9696/Lilu) project and downloaded the latest release. Installed into the `kexts\Others` path.
4. Cheked out the [Lilu Known Plugins list](https://github.com/vit9696/Lilu/blob/master/KnownPlugins.md) and ended up downloading and installing new versions of the following kexts from the links on that page: AppleALC.kext; CoreDisplayFixup.kext; HibernationFixup.kext; IntelGraphicsFixup.kext; Shiki.kext; WhateverGreen.kext.
5. Shutdown and rebooted (rather than a restart, just in case) without the `-liluoff` flag. No problems encountered at all.

So far, everything seems stable but it's only been about 10 minutes since I got this up and running again. Glad to have my Plex server back! :)
